MobiFlight/Joysticks/VKB/VKBEncoder.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System.Collections.Generic;
+using static MobiFlight.Joysticks.VKB.VKBEncoder;
+
+namespace MobiFlight.Joysticks.VKB
+{
+    internal class VKBEncoder
+    {
+        public enum EncoderAction
+        {
+            DEC = 0,
+            INC = 5
+        }
+        private bool firstStart = false; // Ensures initial values get pulled from the device when encoder object is created from definition file.
+        public JoystickDevice DeviceInc = null; // Virtual button for increment events.
+        public JoystickDevice DeviceDec = null; // Virtual button for decrement events.
+        private ushort value = 0; // Stores current state of encoder
+        public VKBEncoder(byte index, ushort? initialValue = null)
+        {
+            // Constructor if virtual button devices are not created otherwise.
+            if (initialValue == null) firstStart = true;
```

Comment:
Please put the `firstStart = true` in `{}`. This applies for all if/else/etc statements in all files touched by the PR. Thanks!